eng/npm/README.md

```diff
@@ -1,15 +1,453 @@
+- 📊 **Azure Cosmos DB** - NoSQL database operations
+- 🧮 **Azure Data Explorer** - Analytics queries and KQL
+- 🐘 **Azure Database for PostgreSQL** - PostgreSQL database management
+- 🐬 **Azure Database for MySQL** - MySQL database management
+- 📊 **Azure Event Grid** - Event routing and management
+- ⚡ **Azure Functions** - Function App management
+- 🧮 **Azure Foundry** - AI model management, AI model deployment, and knowledge index management
+- 🚀 **Azure Managed Grafana** - Monitoring dashboards
+- 🗃️ **Azure Managed Lustre** - High-performance Lustre filesystem operations
+- 🔑 **Azure Key Vault** - Secrets, keys, and certificates
+- ☸️ **Azure Kubernetes Service (AKS)** - Container orchestration
+- 📦 **Azure Load Testing** - Performance testing
+- 🏪 **Azure Marketplace** - Product discovery
+- 📈 **Azure Monitor** - Logging, metrics, and health monitoring
+- 🏥 **Azure Service Health** - Resource health status and availability
+- ⚙️ **Azure Native ISV Services** - Third-party integrations
+- 🛡️ **Azure Quick Review CLI** - Compliance scanning
+- 📊 **Azure Quota** - Resource quota and usage management
+- 🔴 **Azure Redis Cache** - In-memory data store
+- 🏗️ **Azure Resource Groups** - Resource organization
+- 🎭 **Azure RBAC** - Access control management
+- 🚌 **Azure Service Bus** - Message queuing
+- 🗄️ **Azure SQL Database** - Relational database management
+- 🗄️ **Azure SQL Elastic Pool** - Database resource sharing
+- 🗄️ **Azure SQL Server** - Server administration
+- 💾 **Azure Storage** - Blob, table, file, and data lake storage
+- 📋 **Azure Subscription** - Subscription management
+- 🏗️ **Azure Terraform Best Practices** - Infrastructure as code guidance
+- 🖥️ **Azure Virtual Desktop** - Virtual desktop infrastructure
+- 📊 **Azure Workbooks** - Custom visualizations
+- 🏗️ **Bicep** - Azure resource templates
+- 🏗️ **Cloud Architect** - Guided architecture design
+
+## Documentation
+
+- See our [official documentation on learn.microsoft.com](https://learn.microsoft.com/azure/developer/azure-mcp-server/) to learn how to use the Azure MCP Server to interact with Azure resources through natural language commands from AI agents and other types of clients.
+- For additional command documentation and examples, see our [GitHub repository section on Azure MCP Commands](https://github.com/microsoft/mcp/blob/main/docs/azmcp-commands.md).
+
+
+## Feedback & Support
+
+- Check the [Troubleshooting guide](https://aka.ms/azmcp/troubleshooting) to diagnose and resolve common issues with the Azure MCP Server.
+- We're building this in the open. Your feedback is much appreciated, and will help us shape the future of the Azure MCP server.
+    - 👉 Open an issue in the public [GitHub repository](https://github.com/microsoft/mcp/issues) — we’d love to hear from you!
+
+## Contributing
+
+Want to contribute?
+Check out our [contribution guide](https://github.com/microsoft/mcp/blob/main/eng/vscode/CONTRIBUTING.md) to get started.
```

Comment:
this link points to the VS Code VSIX-specific contributing.md.  Please change this to the right CONTRIBUTING.md link